fw/http_frame.c

```diff
@@ -2116,6 +2149,13 @@ do {									\
 						       &stream->xmit.postponed);
```

Comment:
As I think trailers works same as headers frames. So we can't send postponed skbs if we start but not finished sending trailer frames. We also need test this rarely case. We should implement test with a very long trailer. And simulate a situation when client sends us new settings (for example) after receiving first trailer frame but before receiving all trailers.

Reply:
The role of trailers is equivalent to the body. In fact, the only application of HTTP/2 trailers so far is GRPC, which uses the trailers to represent the call result, and the trailers are expected to follow the body from the perspective of GRPC client.

Reply:
I mean that if we have long trailer about 30000 bytes for example we can't send it without continuation frame. If we send some other frame after we send first trailer (in fact headers) frame but before we send continuation trailer (continuation headers) frame we catch connection error of type protocol error. For this purpose we have postponed skbs.

Reply:
I get it. I'll check it later.
A quick question, shouldn't `stream->xmit.frame_length` be non-zero if any continuations? If so, why it will come into this branch?

Reply:
We come to this branch after all continuation frames will be sent

Reply:
Yes, I know. I mean the code should use `*_len` as conditional. The `frame_length` here looks rather obscure. I'll tune it later.

Reply:
It is not correct you should send postponed frames after sending headers frames, when h_len == 0 (before you send body) and after you send trailer frames (t_len == 0). Otherwise you can block all other frames until body and trailers will be send (body can be very long)

Reply:
I don't get your words. Can you make a suggestion change directly? Thanks.